src/routes/targeted-messaging/targeted-messaging.controller.ts

```diff
@@ -0,0 +1,100 @@
+import { TargetedSafeNotFoundError } from '@/domain/targeted-messaging/errors/targeted-safe-not-found.error';
+import {
+  CreateSubmissionDto,
+  CreateSubmissionDtoSchema,
+} from '@/routes/targeted-messaging/entities/create-submission.dto.entity';
+import { Submission } from '@/routes/targeted-messaging/entities/submission.entity';
+import { TargetedMessagingService } from '@/routes/targeted-messaging/targeted-messaging.service';
+import { AddressSchema } from '@/validation/entities/schemas/address.schema';
+import { NumericStringSchema } from '@/validation/entities/schemas/numeric-string.schema';
+import { ValidationPipe } from '@/validation/pipes/validation.pipe';
+import {
+  Body,
+  Controller,
+  DefaultValuePipe,
+  Get,
+  HttpStatus,
+  NotFoundException,
+  Param,
+  ParseIntPipe,
+  Post,
+  Res,
+} from '@nestjs/common';
+import { ApiCreatedResponse, ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { Response } from 'express';
+
+@ApiTags('targeted-messaging')
+@Controller({
+  path: 'targeted-messaging',
+  version: '1',
+})
+export class TargetedMessagingController {
+  constructor(private readonly service: TargetedMessagingService) {}
+
+  @ApiOkResponse({ type: Submission })
+  @Get(
+    'outreaches/:outreachId/chains/:chainId/safes/:safeAddress/signers/:signerAddress/submissions',
```

Comment:
Nit: as these are the same, we can move the prefix to the `path` property of the `Controller` decorator.

Reply:
Done in [3707402](https://github.com/safe-global/safe-client-gateway/pull/1989/commits/37074023fa76e41a4731987b0fe2b5ba31111eef)

Thanks!